aircraft/breguet_range.py

```diff
@@ -0,0 +1,53 @@
+"""Breguet range model"""
+from gpkit.shortcuts import Var, Model
+from gpkit.tools import te_exp_minus1
+
+class BreguetRange(Model):
+
+    """Breguet Range Model
+
+    Assumptions
+    -----------
+    Fuel burn varies linearily with thrust and is independent of velocity.
```

Comment:
You mean in the docstring? Or as `Quantity` objects in the kwargs?

@bqpd, is the design pattern implemented here what you would suggest for models, or would it be better to specify the default values directly in the `Var()` constructors, ans assume they'll be modified by substitution?

I hesitate to require the kwarg units to be specified in the docstring, since they could easily get out of sync if someone changes the units...